### PR TITLE
bigone fetchMarkets

### PR DESCRIPTION
**Pull request by frosty00**
```
 {
    "id": "EOS-USDT",
    "uuid": "feefa85f-99d4-4eda-97c7-8de4c0c45931",
    "symbol": "EOS/USDT",
    "base": "EOS",
    "quote": "USDT",
    "baseId": "EOS",
    "quoteId": "USDT",
    "active": true,
    "precision": {
      "amount": 2,
      "price": 4
    },
    "limits": {
      "amount": {
        "min": 0.01
      },
      "price": {
        "min": 0.00009999999999999999
      },
      "cost": {
        "min": 1
      }
    },
    "info": {
      "id": "feefa85f-99d4-4eda-97c7-8de4c0c45931",
      "name": "EOS-USDT",
      "quote_scale": "4",
      "quote_asset": {
        "id": "17082d1c-0195-4fb6-8779-2cdbcb9eeb3c",
        "symbol": "USDT",
        "name": "TetherUS"
      },
      "base_asset": {
        "id": "b2708373-f732-4ccf-a8e8-e4f860a7e64a",
        "symbol": "EOS",
        "name": "EOS"
      },
      "base_scale": "2",
      "min_quote_value": "1"
    }
  }
]
```

### CODE REVIEW

File: js/bigone.js [hunk 1]
```diff
@@ -192,9 +192,13 @@ module.exports = class bigone extends Exchange {
             const base = this.safeCurrencyCode (baseId);
             const quote = this.safeCurrencyCode (quoteId);
             const symbol = base + '/' + quote;
+            const amountPrecisionString = this.safeString (market, 'base_scale');
+            const pricePrecisionString = this.safeString (market, 'quote_scale');
+            const amountLimit = (amountPrecisionString === undefined) ? undefined : '1e-' + amountPrecisionString;
```
Comment (kroitor) [inline js/bigone.js:197]: What if we introduce a helper base method for this ternary conditional to avoid code clutter?

Comment (frosty00) [reply js/bigone.js:197]: we could... I don't see it to be too critical

Comment (kroitor) [reply js/bigone.js:197]: Also, if precision is negative, it will yield `1e--...`.

Comment (kroitor) [reply js/bigone.js:197]: That's at least two reasons to detach this bit to a base method, what do you say?

Comment (frosty00) [reply js/bigone.js:197]: yeah that is a good point, will consider doing that this week

**Pull request merged**